app/models/school.rb

```diff
@@ -35,6 +35,11 @@ def northing=(northing)
     set_geolocation_from_easting_and_northing
   end
 
+  def has_religious_character?
+    return false if !self.respond_to?(:gias_data) || self.gias_data == nil
```

Comment:
No need to change, but this might be a bit clearer as 

`return false unless self.respond_to?(:gias_data) || self.gias_data == nil`

Reply:
This changes the scope of the negative (in both English and Ruby).

Reply:
This changes the scope of the negative (in both Ruby and English).